frontend/src/components/BackLink/index.module.scss

```diff
@@ -1,11 +1,15 @@
-.backLinkWrapper {
-  margin: 24px 0 92px 24px;
+.buttonWrapper {
+  margin: 24px 0 24px 24px;
 }
 
-.backLink {
+.button {
   width: 32px;
   height: 32px;
+  padding: 0;
   color: var(--color-light-pink);
+  background: none;
+  border: none;
+  outline: inherit;
```

Comment:
Where is the backlink inheriting the outline from? 🤔

Reply:
Was a mistake leftover, just wanted to remove the default button styles. going to fix that.